Code walkthrough (#59)
NVIDIA/nim-anywhere: README.md

to help our partners and customers build effective RAG pipeline with
ease.

NIM Anywhere contains all the tooling required to start integrating NIM
microservices for RAG. It natively scales out to full-sized labs and up
to production environments. This is great news for building a RAG
architecture and easily adding NIM microservices as needed. If you're
unfamiliar with RAG, it dynamically retrieves relevant

> 

Reviewer: I believe this is an accidental newline? Seems like the sentence continues

blog](https://developer.nvidia.com/blog/enhancing-rag-applications-with-nvidia-nim/).

Get started with NIM Anywhere now with the [quick-start](#quick-start)

> instructions and build your first RAG application using NIMs!

Reviewer 2: ```suggestion
instructions and build your first RAG application using NIM Microservices!
```

Reviewer 3: NIMs are Nvidia Inference Microservices.  So an NIM Microservice is a Nvidia Inference Microservice Microservice 

Kind of like ATM Machine would be wrong.

Reviewer 2: Following the convention seen out [here](https://www.nvidia.com/en-eu/ai/?ncid=pa-srch-goog-201921&_bt=728997781338&_bk=nvidia%20ai&_bm=b&_bn=g&_bg=173378669933&gad_source=1&gclid=EAIaIQobChMIpfqSruSViwMVXlz_AR2WbA48EAAYASAAEgKMfvD_BwE) and present in other updated public-facing content.

![NIM Anywhere Screenshot](.static/_static/nim-anywhere.png)
 
- [Quick-start](#quick-start)
  - [Generate your NGC Personal Key](#generate-your-ngc-personal-key)
  - [Install AI Workbench](#install-ai-workbench)
  - [Download this project](#download-this-project)
  - [Configure this project](#configure-this-project)

>   - [Start This Project](#start-this-project)

Reviewer 2: ```suggestion
  - [Start this project](#start-this-project)
```

  - [Populating the Knowledge Base](#populating-the-knowledge-base)

> - [Developing Your Own Applications](#developing-your-own-applications)

Reviewer 2: ```suggestion
- [Developing your own applications](#developing-your-own-applications)
```
